README.md

```diff
@@ -110,10 +206,19 @@ includeBuild('<PATH-TO-RSKJ-SOURCE-CODE>') {
 
 **Note:** Change PATH-TO-RSKJ-SOURCE-CODE value to your local Rskj path.
 
+
+### Steps to run the project.
+
+*To run from command line:*
+
+```bash
+./gradlew run -PmainClass=co.rsk.federate.FederateRunner
```

Comment:
In order to run the powpeg-node from command you will have to also provide the configuration file, please add that to the description here.